common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -64,6 +65,9 @@
             WORKDIR = tmpdir();
             logger.debug("-Dio.netty.native.workdir: " + WORKDIR + " (io.netty.tmpdir)");
         }
+
+        DELETE_NATIVE_LIB_AFTER_LOADING = SystemPropertyUtil.getBoolean(
+                "io.netty.native.deleteLibAfterLoading", true);
```

Comment:
@Scottmitch I can do this but I think either you want it for all or for none. So I think it is overkill and makes not a lot of sense in a practical sense. So please tell me if you feel super strong about it and if so I will do, but honestly I think it makes more sense as it is.